src/Tribe/Service_Providers/Context.php

```diff
@@ -351,7 +383,8 @@ static function ( $data ) {
 								$date_k = 'tribe-bar-date';
 
 								return is_array( $data ) && isset( $data[ $date_k ] )
-									? add_query_arg( [ $date_k => $data[ $date_k ] ], tribe_get_request_var( 'url', home_url() ) )
+									? add_query_arg( [ $date_k => $data[ $date_k ] ],
+										tribe_get_request_var( 'url', home_url() ) )
```

Comment:
* :no_entry_sign: Multi-line function call not indented correctly; expected 36 spaces but found 40
* :no_entry_sign: Closing parenthesis of a multi-line function call must be on a line by itself

<sub>hash: f2831bfd23b42e902edba285e19453bc</sub>